### ISSUE TITLE
Patch French localization properties

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#40]
```
Hello,

please find attached a patch to update the french localization properties.

Cheers,
Olivier Parent
```

Original issue reported on code.google.com by `opar...@gmail.com` on 22 Oct 2008 at 12:19

**Comment by GoogleCodeExporter** [comment]
```
Thanks a lot! The changes will be included in the next release.
Regards,
Thomas
```

Original comment by `thomas.t...@gmail.com` on 23 Oct 2008 at 9:28
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Available in version 1.1.102
```

Original comment by `thomas.t...@gmail.com` on 25 Oct 2008 at 2:25
- Changed state: **Fixed**
